src/callbacks/sirius.js

```diff
@@ -39,15 +37,15 @@ var parameter = ''
  * @type {{name: string, token: string}}
  */
 var user = {
-  name: ReactCookie.load('oauth_nickname'),
+  name: ReactCookie.load('oauth_username'),
   token: ReactCookie.load('oauth_access_token'),
 }
 
 /**
- * Actual Sirius API URL
+ * URL of proxy for Sirius API
  * @type {string}
  */
-var siriusAPIUrl = SIRIUS_BASE_URL
+var siriusAPIUrl = `${getBaseUri()}/_proxy/api/v1/`
```

Comment:
Alternatively there could be an optional environment variable to enable proxying.